bridge/client/app/_components/ktb-keptn-services-list/ktb-keptn-services-list.component.html

```diff
@@ -64,16 +72,33 @@
 
     <ng-container *ngIf="selectedUniformRegistration">
       <h3 class="mt-3 mb-0"><span class="mr-1 bold" [textContent]="selectedUniformRegistration.name"></span></h3>
-      <h3 class="mt-1">Error events</h3>
 
-      <dt-loading-spinner *ngIf="isLoadingLogs"></dt-loading-spinner>
+      <ktb-expandable-tile [alignment]="'left'" class="mb-2" [expanded]="true">
+        <ktb-expandable-tile-header>
+          <h3 class="mt-0 mb-0">Subscriptions</h3>
+        </ktb-expandable-tile-header>
+        <ktb-uniform-subscriptions *ngIf="selectedUniformRegistration.hasSubscriptions(projectName), else noSubscriptions" [uniformRegistration]="selectedUniformRegistration"></ktb-uniform-subscriptions>
+        <ng-template #noSubscriptions>
+          No subscriptions for this integration available
+        </ng-template>
+      </ktb-expandable-tile>
 
-      <div fxLayout="column" class="overflow-y-scroll" *ngIf="!isLoadingLogs">
-        <ktb-uniform-registration-logs *ngIf="uniformRegistrationLogs$ | async as logs"
-                                       [projectName]="projectName"
-                                       [lastSeen]="lastSeen"
-                                       [logs]="logs"></ktb-uniform-registration-logs>
-      </div>
+      <ktb-expandable-tile [alignment]="'left'" [expanded]="true">
```

Comment:
Please add a padding / margin somewhere so that the last tile does not stick to the end of the page.